### PR TITLE
Cleanup initialize scripts and add sh scripts

### PR DESCRIPTION
**Pull request by MattSturgeon**
Quickly put together these changes to the patch scripts, there may or may not be minor errors since I was rushing a little!

---

Cleanup some of the uses of temp directoris using git branching more
effectively.

Add bash equivialents for UNIX

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
+1

File: patch/apply-patch.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# master: vanilla Minecraft code, as generated by the MCP.
+# modded: modified Minecraft code, as generated by the apply-patch script.
+# Before running this script, the "modded" branch must be created with
+# "master" as its source.
+
+cd ../mc
+git checkout modded
+git reset --hard master
+git apply --ignore-space-change --ignore-whitespace ../patch/minecraft.patch
+rem pause
```
Comment (MattSturgeon) [inline patch/apply-patch.sh]: This line should be removed

**Comment by MattSturgeon** [comment]
I've given this a second look over and tidied up a couple little bits, hopefully it should all be ok now.

**Comment by MattSturgeon** [comment]
Any feedback on why this was closed?

**Review by ghost — COMMENTED** [review]
@Alexander01998 thinks support for Mac and Unix like systems are unneeded.

**Comment by Alexander01998** [comment]
I don't want any contributions from Mac users.

**Comment by MattSturgeon** [comment]
As a fan of open source software I choose to use Linux as my primary operating system. While I have a distain for Apple and most of their fanbase I do not feel that that is relavent to this.

This PR consists of two changes:

Firstly it simplifies the existing scripts. To me this is a no brainer simple improvememt.

Secondly it adds UNIX equivialents of those scripts. Since the majority of the commands are the same, this feels like it should be no real burdun. **However if you are really that opposed to including a shell script in your projectm, I am happy to make a new PR with only the changes to the batch scripts.**

Frankly I see no reason to deliberately exclude all UNIX users from contributing to your project. I understand not wishing to actively support systems you do not use, however due to the cross platform nature of java and shell scripts this still seems silly.

**Pull request closed without merging**